config/locales/hyrax.en.yml

```diff
@@ -1132,17 +1142,21 @@ en:
       defaults:
         admin_set_id: Administrative Set
         based_near: Location
+        contributor: Contributor
         creator: Creator
         date_created: Date Created
-        description: Description
+        description: Abstract or Summary
         embargo_release_date: until
         files: Upload a file
+        identifier: Identifier
```

Comment:
remove line 1151 for this and all other languages

Reply:
The identifier line has been removed from all languages